packages/editor/src/editor.ts

```diff
@@ -43,7 +54,7 @@ export class Editor extends FormControlMixin(LitElement) {
     this.#value = value;
 
     if (this.#view) {
-      setHTML(value || '')(this.#view.state, this.#view.dispatch, this.#view);
+      setHTML(value?.toString() || '')(this.#view.state, this.#view.dispatch, this.#view);
```

Comment:
Why do you need `toString` here? It either is a string or it it undefined, according to the typing, and when it's undefined it becomes an empty string.